webapp/packages/plugin-help/src/Shortcuts/SHORTCUTS_DATA.ts

```diff
@@ -102,17 +102,31 @@ export const NAVIGATION_TREE_SHORTCUTS: IShortcut[] = [
 function transformKeys(keyBinding: IKeyBinding): string[] {
   const keys = getCommonAndOSSpecificKeys(keyBinding);
 
-  return keys.map(key => transformModToDisplayKey(key.toLocaleUpperCase().replace(/\+/gi, ' + ')));
+  return keys.map(formatKeysSymbols).map(key => transformModToDisplayKey(key.toLocaleUpperCase().replace(/\+/gi, ' + ')));
+}
+
+function formatKeysSymbols(key: string): string {
+  const KEY_FORMAT_MAP: Record<string, string> = {
```

Comment:
It can be moved outside the function to avoid creating the object each time the function runs, since its content is static